### PR TITLE
:white_check_mark: init tests

### PR DESCRIPTION
**Pull request by fabienjuif**
https://github.com/Zenika/react-workbench/issues/6

- All spec files are created with minimum information, so the coverage is accurate

### CODE REVIEW

**Comment by fabienjuif** [comment]
`23.929%` snif

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 /test
 /tmp
+coverage/
```
Comment (bpetetot) [inline .gitignore]: harmonize folders declaration in .gitignore file

File: misc/__mocks__/component.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default (name) => {
+  const debug = {
+    name,
+    props: {},
+  }
+
+  return props => {
```
Comment (bpetetot) [inline misc/__mocks__/component.jsx]: the linter asks for parenthesis around `props`

**Review by bpetetot — DISMISSED** [review]
You can also add `tests status` and `coverage` badges into readme file

File: .gitignore (hunk 1 above)
Comment (fabienjuif) [inline .gitignore]: You asked to do it Friday :) https://github.com/Zenika/react-workbench/issues/58

**Comment by fabienjuif** [comment]
You asked to do it Friday :) https://github.com/Zenika/react-workbench/issues/58

**Review by bpetetot — APPROVED** [review]
👍

**Pull request merged**